.github/workflows/java-ci.yml

```diff
@@ -95,7 +95,7 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        jvm: [11, 17, 21]
+        jvm: [17, 21]
```

Comment:
what's the reason for removing JDK 11 from this? we're releasing JDK 11 jars so I think we should keep testing with JDK 11

Reply:
https://github.com/apache/iceberg/pull/12494/files#r2045335206